dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -696,6 +696,10 @@ error.no.serialfilteragent=CWWKE0949E: The specified serial filter agent jar fil
 error.no.serialfilteragent.explanation=The specified serial filter agent jar file does not exist, or fail to read.
 error.no.serialfilteragent.useraction= Ensure that the file exists and can be read by the current user.
 
+error.package.extension=CWWKE0950E: A --archive option with a value not ending with a .jar file extension with --include=runnable is not an allowed combination.
+error.package.extension.explanation=The --archive option must include a file extension that ends with .jar when specifying --include=runnable.
+error.package.extension.useraction=Supply a file name to --archive with a file extension of .jar when specifying --include=runnable.
```

Comment:
This sentence is a little confusing. What do you mean when you say to supply a file name to --archive?

Reply:
The --archive option gets a parameter which contains the file name to use for the archive path/name:  --archive=c:\myserver.zip  and what we're trying to throw an error message for is when they do an invalid combination with --include=runnable which can only create a archive name that is a .jar file:
--archive=c:\myserver.zip --include=runnable <-- should result in this new error message being displayed

Reply:
It seems confusing to me too.. maybe something like "The archive file name must end with .jar when the runnable option is used"?

Reply:
Ok, I updated the base message with Brents suggestion, and left the explain and useraction the same as they seemed okay to me.

Reply:
Checked with Brandy and Brent - both agreed the new update was ok.